pocketbase/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: pocketbase_app_1
+      APP_PORT: 8090
+      PROXY_AUTH_ADD: "false"
+  
+  app:
+    image: ghcr.io/muchobien/pocketbase:0.22.11@sha256:04dd9850554a24f82cfee060ff394179baaecc0f97e7721d9dd06818d26fd671
+    restart: on-failure
+    stop_grace_period: 1m
+    volumes:
+      - ${APP_DATA_DIR}/data/data:/pb_data
+      - ${APP_DATA_DIR}/data/public:/pb_public
+      - ${APP_DATA_DIR}/data/hooks:/pb_hooks
```

Comment:
Can you please add these 3 host paths with a `.gitkeep` in each as described here (https://github.com/getumbrel/umbrel-apps/pull/1083#discussion_r1609144592)?